src/lib/useRouter.js

```diff
@@ -53,7 +53,7 @@ export default function useRouter({
       if (size <= 1) {
         // if we are at the root, pass this event to our parent
         if (oldPopState.current) {
-          window.history.back();
+          oldPopState.current();
```

Comment:
note: the reason for this change is that when we click the "return" button (which purposefully pops a route) we also inject a new `pushRoute` entry, which means the back button stops working. so instead of using the browser back button (which was a dumb shortcut, anyway), we ask the parent router to pop itself directly.